airflow/www/views.py

```diff
@@ -1760,6 +1761,7 @@ def trigger(self, session=None):
                 'airflow/trigger.html',
                 dag_id=dag_id,
                 origin=origin,
+                html_attr_escaped_origin=html.escape(origin, quote=True),
```

Comment:
I don't see where this value is used in templates? Have I just missed it?

Reply:
You didn't miss it, I just messed up my rebase. Fixing now.